packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageVariables.ts

```diff
@@ -78,8 +76,6 @@ export const chatMessageVariables = (siteVars): ChatMessageVariables => ({
   reactionGroupMarginLeft: pxToRem(12),
   reactionGroupBorderColor: 'transparent',
   showActionMenu: undefined,
-  timestampColor: undefined,
-  timestampColorMine: siteVars.colorScheme.default.foreground1,
```

Comment:
Since we are using `<Text timestamp/>`, my intention was to fall back on that color and not have it be overwritten by ChatMessage styles.

As you can see from the screener, for the v1 themes only the dark theme is impacted, which is intentional as the old color does not have enough contrast.